extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -52,6 +55,21 @@ const MIN_INTEGER = -100000;
  */
 export let panningMediaPositionDef;
 
+/**
+ * Max distances to keep image in viewport.
+ * left: Percentage between [-50; 50]
```

Comment:
If these are percentages, can we reflect that in the `typeDef` name or in the properties? e.g. `leftPercentage`.